### PR TITLE
work on pickup & savegame

### PR DESCRIPTION
**Pull request by zdimension**

### CODE REVIEW

File: GAME/CAMERA.C [hunk 1]
```diff
@@ -700,4 +700,34 @@ void BinocularCamera(struct ITEM_INFO* item)
 void ConfirmCameraTargetPos()
 {
 	S_Warn("[ConfirmCameraTargetPos] - Unimplemented!\n");
-}
\ No newline at end of file
+}
+
+void ScreenShake(struct ITEM_INFO *item, short MaxVal, short MaxDist)
```
Comment (Gh0stBlade) [inline GAME/CAMERA.C]: `struct ITEM_INFO *item -> struct ITEM_INFO* item`

File: GAME/CODEWAD.C [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#include "CODEWAD.H"
+
+#include "TYPES.H"
```
Comment (Gh0stBlade) [inline GAME/CODEWAD.C]: Includes should be ordered alphabetically.

File: GAME/DELTAPAK.C [hunk 3]
```diff
@@ -22,6 +22,7 @@
 #include "LOT.H"
 #include "LARA2GUN.H"
 #include "LARA1GUN.H"
+#include "CODEWAD.H"
```
Comment (Gh0stBlade) [inline GAME/DELTAPAK.C]: Includes should be ordered alphabetically.

File: GAME/SAVEGAME.C [hunk 4]
```diff
@@ -54,14 +59,106 @@ void SaveLevelData(int FullSave)//53AAC, 53F10
...
+	if (!FullSave)
+	{
+		savegame.Lara.item_number = lara.item_number;
+		if (savegame.Lara.IsMoving)
+		{
+			savegame.Lara.IsMoving = 0;
+			savegame.Lara.gun_status = 0;
+		}
+	}
+	memcpy(&lara, &savegame.Lara, sizeof(lara));
```
Comment (Gh0stBlade) [inline GAME/SAVEGAME.C]: Should be `sizeof(struct lara)`

Comment (zdimension) [reply GAME/SAVEGAME.C]: lara is a variable

**Pull request merged**